koku/masu/external/kafka_msg_handler.py

```diff
@@ -328,6 +329,8 @@ def extract_payload(url, request_id, context={}):  # noqa: C901
     report_meta["provider_uuid"] = account.get("provider_uuid")
     report_meta["provider_type"] = provider_type
     report_meta["schema_name"] = schema_name
+    report_meta["account"] = schema_name[4:]
```

Comment:
This is for `acct1234` -> `1234`?  I guess this won't ever change as long as we don't touch `create_schema_name`

Reply:
🤦 I just realized this isn't actually part of the change, I was expanding the view to see more of the function.  Sorry!

Reply:
That value is only used in the context logging and is to highlight a problem where sources isn't giving use data. I can allow it to be `None`, but I don't think it will log well. `schema_name` should always have a value with `acct` prefix in this case or we won't have found a matching provider.

Reply:
Feel free to resolve if you are ok with the above.